akka-docs/src/main/paradox/classic/supervision.md

```diff
@@ -0,0 +1,256 @@
+which will shut down the whole actor system.
+
+### `/`: The Root Guardian
+
+The root guardian is the grand-parent of all so-called “top-level” actors and
+supervises all the special actors mentioned in @ref:[Top-Level Scopes for Actor Paths](../general/addressing.md#toplevel-paths) using the
+`SupervisorStrategy.stoppingStrategy`, whose purpose is to terminate the
+child upon any type of `Exception`. All other throwables will be
+escalated … but to whom? Since every real actor has a supervisor, the
+supervisor of the root guardian cannot be a real actor. And because this means
+that it is “outside of the bubble”, it is called the “bubble-walker”. This is a
+synthetic `ActorRef` which in effect stops its child upon the first sign
+of trouble and sets the actor system’s `isTerminated` status to `true` as
+soon as the root guardian is fully terminated (all children recursively
+stopped).
+
+
+## One-For-One Strategy vs. All-For-One Strategy
+
+There are two classes of supervision strategies which come with Akka:
+`OneForOneStrategy` and `AllForOneStrategy`. Both are configured
+with a mapping from exception type to supervision directive (see
+[above](#supervision-directives)) and limits on how often a child is allowed to fail
+before terminating it. The difference between them is that the former applies
+the obtained directive only to the failed child, whereas the latter applies it
+to all siblings as well. Normally, you should use the
+`OneForOneStrategy`, which also is the default if none is specified
+explicitly.
+
+The `AllForOneStrategy` is applicable in cases where the ensemble of
+children has such tight dependencies among them, that a failure of one child
+affects the function of the others, i.e. they are inextricably linked. Since a
+restart does not clear out the mailbox, it often is best to terminate the children
+upon failure and re-create them explicitly from the supervisor (by watching the
+children’s lifecycle); otherwise you have to make sure that it is no problem
+for any of the actors to receive a message which was queued before the restart
+but processed afterwards.
+
+Normally stopping a child (i.e. not in response to a failure) will not
+automatically terminate the other children in an all-for-one strategy; this can
+be done by watching their lifecycle: if the `Terminated` message
+is not handled by the supervisor, it will throw a `DeathPactException`
+which (depending on its supervisor) will restart it, and the default
+`preRestart` action will terminate all children. Of course this can be
+handled explicitly as well.
+
+Please note that creating one-off actors from an all-for-one supervisor entails
+that failures escalated by the temporary actor will affect all the permanent
+ones. If this is not desired, install an intermediate supervisor; this can very
+be done by declaring a router of size 1 for the worker, see
+@ref:[Routing](../routing.md).
+
+
+## Delayed restarts for classic actors
```

Comment:
strange that this is documented in the concepts page instead of `fault-tolerance.md`

Reply:
Moved